runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixPlugin.java

```diff
@@ -58,19 +72,33 @@
 	@Inject
 	private TearsOfGuthixOverlay overlay;
 
+	@Inject
+	private TearsOfGuthixRotationOverlay rotationOverlay;
+
+	@Provides
+	TearsOfGuthixConfig provideConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(TearsOfGuthixConfig.class);
+	}
+
 	@Getter
 	private final Map<DecorativeObject, Instant> streams = new HashMap<>();
+	private static String rotation = null;

```

Comment:
`null` is the default value for non-primitive fields and can be removed
```suggestion
	private static String rotation;
```